src/main/resources/locales/en/translation/appoint-legal-representative.json

```diff
@@ -1,6 +1,12 @@
 {
   "title": "Appoint a legal representative",
   "p1": "You have chosen to appoint a legal representative so you will need to give the following information to your solicitor so they can perform a Notice of Change.",
+  "h2": "Your claim details",
```

Comment:
Is "claim" a valid word here? 
I know it is copied from SYA. Just want to confirm about this